app/models/question.rb

```diff
@@ -1,5 +1,6 @@
 class Question < ApplicationRecord
   belongs_to :subject
+  belongs_to :user
```

Comment:
chị nghĩ nên custome lại tí cho khỏi nhầm nhỉ
```suggestion
  belongs_to :creator, class_name: User.name, foreign_key: :user_id
```